nucypher/cli/commands/stake.py

```diff
@@ -1276,35 +1282,56 @@ def preallocation(general_config: GroupGeneralConfig,
 @group_staker_options
 @option_config_file
 @option_event_name
+@option_csv
+@option_csv_file
 @group_general_config
-def events(general_config, staker_options, config_file, event_name):
-    """See blockchain events associated to a staker"""
+def events(general_config, staker_options, config_file, event_name, csv, csv_file):
+    """View blockchain events associated with a staker"""
 
     # Setup
     emitter = setup_emitter(general_config)
-    STAKEHOLDER = staker_options.create_character(emitter, config_file)
 
+    if not event_name:
+        raise click.BadOptionUsage(message='You must specify an event name with --event-name')
+        # TODO: Doesn't work for the moment
```

Comment:
Track this TODO with an issue number?

Reply:
@cygnusv is this TODO still applicable? It was added via https://github.com/nucypher/nucypher/commits/a51f675fdacecb78a0f91102cd0dc28a4ae18aef.